static/sass/_pattern_buttons.scss

```diff
@@ -13,4 +13,17 @@
   .p-strip--accent .p-button--neutral .p-link--external {
     color: $color-dark;
   }
+
+  .p-button--brand,
```

Comment:
@anthonydillon I think it would be cleaner to do a `s/p-button--brand/p-button--positive/g` rather than override the Vanilla p-button--brand... I assume we will have to clean up wrong html for a while in reviews, but better to leave Vanilla as is.